_credentials/quotient.js

```diff
@@ -17,7 +17,7 @@ export function createBankIdCredential({
       distribute: true,
       credential: {
         id: `https://creds-testnet.dock.io/${uuidv4()}`,
-        schema: 'https://schema.dock.io/QuotientBankIdentity-V4-1708715398120.json',
+        schema: 'https://schema.truvera.io/QuotientBankIdentity-V4-1747940495670.json',
```

Comment:
not needed for this PR, but should these be env vars?

Reply:
yes I agree. I can push a PR for that, will update our bank-demo too